pkg/statements/sql/table_constraints.sql

```diff
@@ -10,5 +10,6 @@ JOIN
 WHERE
   n.nspname = $1
   AND relname = $2
+  AND contype != 'n'
```

Comment:
To keep the list of constraints consistent and backwards compatible with other versions of PG